scripts/lint.sh

```diff
@@ -39,8 +39,8 @@ function golanglinter() {
     # lint e2e tests go code
     go install -v github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}
 
-    echo "Linting Teleporter E2E tests Golang code..."
-    cd $TELEPORTER_PATH/tests
+    echo "Linting Golang code..."
+    cd $TELEPORTER_PATH/
```

Comment:
It looks like this will apply linting to all Go files in the repository, including the ABI generated ones. Can you filter those from the linter?

Reply:
this is already filtered out by `.golangci.yml`, it ignores the abi gen directory

Reply:
Nice, didn't realize that